docs/docs/index.md

```diff
@@ -23,24 +25,16 @@ At [H2O Health](https://h2o.ai/solutions/industry/health/), we build AI apps tha
 
 <meta name="viewport" content="width=device-width, initial-scale=1"></meta>
 
-<div class="row">
-
-  <div class="column">
-    <div class="card">
-      <h2><Icon>rocket_launch</Icon> Get started</h2>
-  <div class="space">
-
-[What is H2O Health?](./get-started/what-is-h2o-health/what-is-h2o-health.md)
-
-  </div>
-  <div class="space">
-
-[Access H2O Health](./get-started/access-h2o-health-apps/access-h2o-health-apps.md)
-
-  </div>
-    </div>
-  </div>
-
+<div class="home-suite-container">
+  <a href="epidemiology/covid-19-hospital-occupancy-simulator/overview" class="home-app-link">
+    <img src={chos} alt="" width="150"/> <br/>
+    <span>Covid-19 Hospital Occupancy Simulator (CHOS)</span>
+  </a>
+  <br/>
+  <a href="gene-mutation/overview" class="home-app-link">
+    <img src={geneMutation} alt="" width="100"/> <br/>
+    <span>Gene Mutation AI</span>
+  </a>
```

Comment:
@oshi98 Please position the logos in a horizontal way as its currently structure here: https://docs.h2o.ai/h2o-health/, also, please add the navigation section under this line